### PR TITLE
Extend consumer parcel

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/BuildingRegistry.Consumer.Read.Parcel/Migrations/20230913140359_AddSpatialIndexOnGeometry.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+           LEVEL_1 = MEDIUM,
+           LEVEL_2 = MEDIUM,
+           LEVEL_3 = MEDIUM,
+           LEVEL_4 = MEDIUM),
+         CELLS_PER_OBJECT = 5)");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
```
Comment (pgallik) [inline src/BuildingRegistry.Consumer.Read.Parcel/Migrations/20230913140359_AddSpatialIndexOnGeometry.cs:27]: migrationBuilder.Sql(@"
                DROP INDEX [SPATIAL_ParcelItems_Geometry] ON [{Schema.ConsumerReadParcel}].[{ParcelConsumerItemConfiguration.TableName}]
                GO");

**Review by pgallik — DISMISSED** [review]

File: src/BuildingRegistry.Consumer.Read.Parcel/Migrations/20230913140359_AddSpatialIndexOnGeometry.cs (hunk 1 above)
Comment (ArneD) [reply src/BuildingRegistry.Consumer.Read.Parcel/Migrations/20230913140359_AddSpatialIndexOnGeometry.cs:27]: What @pgallik says, without `GO`

**Comment by ArneD** [comment]
Don't forget the tests: https://github.com/Informatievlaanderen/building-registry/blob/main/test/BuildingRegistry.Tests/ProjectionTests/Consumer.Parcel/ParcelConsumerKafkaProjectionTests.cs

**Review by pgallik — APPROVED** [review]

**Pull request merged**